src/main/resources/assets/farmersdelight/lang/es_es.json

```diff
@@ -62,30 +67,32 @@
   "item.farmersdelight.hot_cocoa": "Chocolate caliente",
   "item.farmersdelight.wheat_dough": "Masa de trigo",
   "item.farmersdelight.raw_pasta": "Pasta cruda",
-  "item.farmersdelight.pumpkin_slice": "Pedazo de calabaza",
-  "item.farmersdelight.cabbage_leaf": "Hoja de repollo",
-  "item.farmersdelight.minced_beef": "Carne molida",
-  "item.farmersdelight.beef_patty": "Bife de hamburguesa",
-  "item.farmersdelight.chicken_cuts": "Pedazos de pollo crudo",
-  "item.farmersdelight.cooked_chicken_cuts": "Pedazos de pollo cocido",
+  "item.farmersdelight.pumpkin_slice": "Rodaja de calabaza",
+  "item.farmersdelight.cabbage_leaf": "Hoja de col",
+  "item.farmersdelight.minced_beef": "Carne picada",
+  "item.farmersdelight.beef_patty": "Hamburguesa cruda",
```

Comment:
Beef Patties are actually the cooked form; the term "cruda" (raw) might be wrong here.

Reply:
sorry :7

Reply:
@vectorwing fixed